config/routes.rb

```diff
@@ -4,4 +4,9 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
+
+  resources :posts, only: [:index, :show, :new, :create] do
+    resources :comments, only: [:new, :create]
+    resources :likes, only: [:create]
+  end
 end
```

Comment:
- [ ] There is an issue when saving a new post from the current user to the database and routing error appears on the console. Check this and try to combine these lines on one outer user block, here's an example: 
![image](https://github.com/Munyabelden/Blog_App/assets/91486791/db1dc92d-60ae-4b1b-9679-221e6d4e60a8)
```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, new, create] do
      resources :comments, only: [:new, :create]
      resources :likes, only: [:create]
    end
  end
```